dataset.py

```diff
@@ -247,15 +247,15 @@ def partitioned_reference_data(reference_csv_filename: str) -> PartitionedDatase
   partitioned_dataset.print_split(partition_data)
   return partition_data
 
-def load_reference_samples(filters: list[ee.Filter] = []) -> pd.DataFrame:
+def load_reference_samples(org_name , filters: list[ee.Filter] = []) -> pd.DataFrame:
   '''
   Given an optional list of filters, returns a DataFrame containing all current
   reference sample from Earth Engine. You must have the proper authorization
   to access this data, which is obtained by belonging to an organization added
   to TimberID.org
   '''
   eeddf.initialize_ddf()
-  fc = ee.FeatureCollection('projects/river-sky-386919/assets/timberID/trusted_samples')
+  fc = ee.FeatureCollection('projects/river-sky-386919/assets/ee_org/' + org_name +' /trusted_samples')
```

Comment:
we need to support options for prod environment as well as org.
Kader can we just make the change here now so that it should just work next week when we set up prod?  
Should be:

projects/timberid-prd/assets/ee_org/{org_name}/trusted_samples
right?

Reply:
that's true, projects/timberid-prd/assets/ee_org/{org_name}/trusted_samples

Reply:
can you make an option that uploads to that location?  An additional argument?

Reply:
done!